RT56 Turkey Last Major Update

BiccyThiccy opened the pull request:
- Every political path has been fully reworked from the ground up.
- Localization improvements: the couple of Turkish names have been deleted, I deleted all of the localization collides I could find, and made some improvements to existing stuff.
- Added Thrace as a coreable state (same decision menu as the Azerbaijani stuff).
- Other misc crap in the focus tree, too much to list really.
- RT56 Tree now starts with no cores on Kurdish states, and modifiers akin to what Sudan has.

### For the patch-notes:
**Highlights:**

- The rework to the Road to 56 Turkish Focus Tree has officially been completed, reworking the political paths from the ground up, and breathing life into other areas of the tree.

BiccyThiccy committed:
Focus tree layout

BiccyThiccy committed:
February 19 push

BiccyThiccy committed:
Update February 21

BiccyThiccy committed:
Update based on testing

BiccyThiccy committed:
Update 1 February 22nd

BiccyThiccy committed:
More localization, small focus tweaks

BiccyThiccy committed:
Commit #1 February 23

BiccyThiccy commented:
Needs before I mark ready for review:

- Localize the remaining stuff
- Get feedback
- See if there's any last minute additions I can make

as far as I can tell, error log is clean

BiccyThiccy committed:
Remaining localization

BiccyThiccy committed:
AI and AI-related changes based on testing

BiccyThiccy committed:
FOCUS_FILTERS, one needed check

BiccyThiccy committed:
Most of the suggested changes

I've marked every suggested change I've implemented with ✅

BiccyThiccy committed:
More Norman+Fantom suggested

Comes with a few of my own improvements as well

BiccyThiccy committed:
Update Feb 25

BiccyThiccy committed:
Loc improvements, day counters for focus flags

BiccyThiccy committed:
Last Update February 26

BiccyThiccy committed:
READY FOR REVIEW

BiccyThiccy committed:
One category I forgot to add

BiccyThiccy committed:
Update from Beta Edits

BiccyThiccy committed:
Last of the suggested changes

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Mar2024-Turkey

SwedishNorwegian committed:
Update TUR_decision_categories.txt

BiccyThiccy committed:
.

BiccyThiccy committed:
Merge branch 'Mar2024-Turkey' of https://github.com/deliciousmods/1956_beta into Mar2024-Turkey

BiccyThiccy committed:
Balkan Pact modifier + removing the KEMchampions

BiccyThiccy committed:
No more Cakmak for Ottoman/Commie paths

marijn211 committed:
Merge branch 'Beta-Edits' into Mar2024-Turkey

marijn211 reviewed (commented):
Dont deviate formatting standards, unless you had a good reason

marijn211 committed:
Event formatting

marijn211 committed:
Fixed error log spamming thing

marijn211 reviewed (dismissed):
Rename event options not following a/b/c/d/e naming format, make sure leader events dont create duplicate leader roles like Ismet Inonu now gets

BiccyThiccy committed:
Marijn Requested + one post-AAT bug fix

BiccyThiccy committed:
Event Loc File

BiccyThiccy committed:
Tweaks based on test run (Description)

- Worsened 'Traumatized Nation' and starting Kurdish debuffs
- Buffed several eco focuses
- Buffed the Village Institute Ideas
- RT56-tree specific tweaks to some BFTB events
- Tweaked the Bicontinental Intervention branch a bit

BiccyThiccy committed:
.

BiccyThiccy committed:
Small issue fix

TheBrit7 commented:
Seems good, just needs approval from Marijn.

ilkober commented on common/scripted_triggers/00_scripted_triggers.txt:
NOT = { AND = { can be replaced

ilkober commented on events/BFTB_Turkey.txt:
No base value in ai_chance meaning that factor will not have an impact there

ilkober commented on events/Turkey.txt:
Same here

Should be like:
ai_chance = {
       base = X
       modifier = {
             factor = 100
             [YOUR CONDITIONS]
       }
}

ilkober commented on interface/replace/r56_goals_shine.gfx:
Two icons are defined, while the new shine is defined for only one means that unless it is defined elsewhere, the shine will not appear while cult_of_the_marshal is selected

BiccyThiccy replied:
Shine works fine in game as far as I can tell, please check the video I linked in #burkey-turkey

BiccyThiccy committed:
ai_chance as requested

BiccyThiccy committed:
Merge branch 'Beta-Edits' into Mar2024-Turkey

ilkober reviewed (approved):
Merging time
Looks good

BiccyThiccy commented:
🫡
